src/imitation/policies/replay_buffer_wrapper.py

```diff
@@ -0,0 +1,83 @@
+"""Wrapper for reward labeling for transitions sampled from a replay buffer."""
+
+
+from typing import Mapping, Optional, Type
+
+import numpy as np
+import torch as th
+from stable_baselines3.common.buffers import ReplayBuffer
+from stable_baselines3.common.type_aliases import ReplayBufferSamples
+
+from imitation.rewards.reward_function import RewardFn
+from imitation.util import util
+
+
+def _samples_to_reward_fn_input(
+    samples: ReplayBufferSamples,
+) -> Mapping[str, np.ndarray]:
+    """Convert a sample from a replay buffer to a numpy array."""
+    return dict(
+        state=samples.observations.cpu().numpy(),
+        action=samples.actions.cpu().numpy(),
+        next_state=samples.next_observations.cpu().numpy(),
+        done=samples.dones.cpu().numpy(),
+    )
+
+
+class ReplayBufferRewardWrapper(ReplayBuffer):
+    """Relabel the rewards in transitions sampled from a ReplayBuffer."""
+
+    def __init__(
+        self,
+        *args,
+        replay_buffer_cls: Type[ReplayBuffer] = ReplayBuffer,
```

Comment:
SB3 calls this `replay_buffer_class` -- use that as well for consistency? https://stable-baselines3.readthedocs.io/en/master/_modules/stable_baselines3/common/off_policy_algorithm.html

Reply:
Agreed! Thanks!